README.md

```diff
@@ -43,6 +43,10 @@ Following mechanisms may be used for DUT control:
 
 ![regression-architecture](https://cloud.3mdeb.com/index.php/s/KkERgGoniBtjfC4/preview)
 
+### Current osfv_cli purpose
+
+![](./docs/img/osfv_cli_after.png)
```

Comment:
Można dać ze dwa zdania kontekstu pod tym.

Że nie używamy obecnie repozytoriów takich jak:
- https://github.com/3mdeb/sonoff-rest-api
- https://github.com/3mdeb/rtectrl-rest-api

Zamiast tego, używamy wspólnych bibliotek z: https://github.com/Dasharo/osfv-scripts/tree/main/osfv_cli/src/osfv/libs

Są one używane jednocześnie w `osfv_cli` jak i testach OSFV.

Biblioteki RF korzystające ze wspólnych keywordów są tutaj: https://github.com/Dasharo/osfv-scripts/tree/main/osfv_cli/src/osfv/rf

A wszystko to do środowiska dołączane jest przez `requirements.txt` - należy więc dbać o jego aktualność.

Reply:
info included: https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/ede9434d7fa2744836168f82ed751eae806ecb54

Reply:
@wiktormowinski please fix according to offline discussion

Reply:
sure thing: https://github.com/Dasharo/open-source-firmware-validation/pull/459/commits/b80963d585c0baafde69e70b70fe9c8d5682dcc8